src/Security/Authorization/Core/src/AuthorizationBuilder.cs

```diff
@@ -29,7 +29,7 @@ public AuthorizationBuilder(IServiceCollection services)
     /// Defaults to true.
     /// </summary>
     /// <returns>The builder.</returns>
-    public virtual AuthorizationBuilder SetInvokeHandlersAfterFailure(bool invoke)
+    public virtual AuthorizationBuilder InvokeHandlersAfterFailure(bool invoke)
```

Comment:
Set* seems better tbh

Reply:
Okay, going back to the original SetInvokeHandlersAfterFailure